packages/trip-form/src/SettingsSelectorPanel.story.tsx

```diff
@@ -92,8 +92,8 @@ const SettingsPanelTemplate = (args: StoryArgs, context: StoryContext) => {
         queryParamMessages={
           args.useCustomMessages
             ? {
-                walkSpeed: {
-                  label: "Custom walk speed label",
+                maxWalkDistance: {
+                  label: "Custom max walk distance label",
```

Comment:
Didn't `maxWalkDistance` get removed from OTP?

Reply:
Yes, it absolutely did, but this is just demoing the ability of this unrelated component to display custom labels. You're right that it doesn't really make sense, but updating this other component's story isn't really within the scope of this PR.